routes/helpers/localizeResults.js

```diff
@@ -0,0 +1,24 @@
+module.exports = (locale, dbResults) => {
+
+  // if currentLocale === defaultLocale, do nothing
+  // english is default
+  if (locale === 'en') return dbResults;
+
+  // keynames in DB that contain translatable content follow pattern 'name___locale'
+  const separator = '___';
+  const translatableKeyReg = /\_{3}.+$/;
+
+  // gets array of translatable values from DB results
+  const translatableFields = Object.keys(dbResults._doc)
```

Comment:
yes, I believe so. But I'm still learning about mongoose. If not, we'll soon find out!